modules/serverless-rn-1-19-0.adoc

```diff
@@ -0,0 +1,58 @@
+[id="serverless-rn-1-19-0_{context}"]
+= Release Notes for Red Hat {ServerlessProductName} 1.19.0
+
+[id="new-features-1-19-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 0.25.1.
+* {ServerlessProductName} now uses Knative Eventing 0.25.0.
+* {ServerlessProductName} now uses Kourier 0.25.0.
+* {ServerlessProductName} now uses Knative `kn` CLI 0.25.1.
+* {ServerlessProductName} now uses Knative Kafka 0.25.0.
+* The `kn func` CLI plug-in now uses `func` 0.19.0.
+
+* The `KafkaBinding` API is deprecated in {ServerlessProductName} 1.19.0 and will be removed in a future release.
+
+* HTTPS redirection is now supported and can be configured either globally for a cluster or per each Knative service. See the Knative documentation about link:https://knative.dev/development/serving/services/http-option/[HTTPS redirection].
+
+[id="fixed-issues-1-19-0_{context}"]
+== Fixed issues
+
+[id="known-issues-1-19-0_{context}"]
+== Known issues
+
+* In `func` version 0.19.0, some runtimes might be unable to build a function using by using podman. You might see an error message similar to the following:
```

Comment:
```suggestion
* In `func` version 0.19.0, some runtimes might be unable to build a function using podman. You might see an error message similar to the following:
```

Reply:
"by using" is actually something that's required by the IBM style guide 🙂 https://www.ibm.com/docs/en/ibm-style?topic=grammar-prepositions
There are some situations where I'm not that fond of it either 😄 but that's the rules we go by

Reply:
@abrennan89 I think that @lance is referring to `using by using`.

Reply:
@matejvasek just realized that, thanks